components/Books/index.tsx

```diff
@@ -13,9 +13,9 @@ export const Books: FC<IBooks> = ({books, handleBookClick}) => {
       <Layout>
         <section>
           {
-            records.map((book: any) => {
-              return <Book key={book.id}
-                bookId={book.id}
+            records.map((book: IBook) => {
```

Comment:
in Line 7, why are you parsing then stringifying the books? 🤔 

```
  const records = JSON.parse(
    JSON.stringify(books)
  )
```

Why can't you use `books` instead?

Reply:
I just realized that you are using `fetch` and not axios. So you need `JSON.stringify(books)`

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#uploading_json_data